check_tv.py

```
@@ -84,194 +87,195 @@ def find_gmail_title(title):
+                    soup = BeautifulSoup(response.text, 'html.parser')
+                    soup.find("div", {"class": "web-player-ending-panel"})
+                    fewtimes += 1
+                    if fewtimes == 6:
+                        logging.info("Stream offline detected. Reloading program...")
+                        if config.unliststream == "True":
+                            logging.info("public back the stream")
+                            public_stream(live_url)
+                        subprocess.run(["taskkill", "/f", "/im", config.apiexe])
+                        subprocess.Popen(["start", "python", "check_tv.py", spare_link, important], shell=True)
+                        break
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)